repose-aggregator/docs/src/docker/Python3Alpine/Dockerfile

```diff
@@ -0,0 +1,16 @@
+FROM python:3-alpine
```

Comment:
We are opting not to use this Dockerfile, right? Is there any reason to keep it around?

Reply:
I don't think there is for the long term.

Reply:
> Also, we probably don't need the `_stylesheets` directory either.

This one is still needed since Ascii Binder just call Ascii Doctor.